src/Logger/Adapter/Sentry.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+namespace Utopia\Logger\Adapter;
+
+use Exception;
+use Utopia\Logger\Adapter;
+use Utopia\Logger\Log;
+use Utopia\Logger\Logger;
+
+// Reference Material
+// https://develop.sentry.dev/sdk/event-payloads/
+
+class Sentry extends Adapter
+{
+    /**
+     * @var string (required, this part of Sentry DSN: 'https://{{THIS_PART}}@blabla.ingest.sentry.io/blabla')
+     */
+    protected string $sentryKey;
+
+    /**
+     * @var string (required, this part of Sentry DSN: 'https://blabla@blabla.ingest.sentry.io/{{THIS_PART}}')
+     */
+    protected string $projectId;
+
+    /**
+     * Return unique adapter name
+     *
+     * @return string
+     */
+    public static function getAdapterName(): string
```

Comment:
```suggestion
    public static function getName(): string
```

The scope already suggests this is the adapter name.

Reply:
Renamed all to `getName`.